pkg/operator/configobserver/featuregates/observe_featuregates_test.go

```diff
@@ -73,6 +73,7 @@ func TestObserveFeatureFlags(t *testing.T) {
 				"CSIMigrationOpenStack=true",
 				"CSIMigrationGCE=true",
 				"CSIMigrationAzureDisk=true",
+				"ExternalCloudProvider=true",
```

Comment:
We need to filter out this feature gate as it's not an upstream FG

Reply:
Actually, I think this is ok. We have to pass the blacklist at the component level (ie operator to remove from operand list).

We expect the FG to be included in the featuregates, the tests there cover that blacklists exclude correctly, so I'm confident if I pass a blacklist into https://github.com/openshift/cluster-kube-controller-manager-operator/blob/6d114f8[…]servation/configobservercontroller/observe_config_controller.go that it will do the right thing

Reply:
I don't think this test should or can care about upstream or not upstream. Note that others in this list aren't upstream FG either (e.g. CSIDriverVSphere).

Reply:
Agreed, we will make sure the upstream vs downstream is handled at the call site